compiler-rt/lib/msan/tests/msan_test.cpp

```diff
@@ -4882,7 +4882,7 @@ TEST(MemorySanitizer, throw_catch) {
   }
 }
 
-#if defined(__linux__)
+#if defined(__GLIBC__)
```

Comment:
Oh, was that just a test?

Reply:
what do you mean? because I changed the interceptor to `SI_GLIBC` (the headers are a mess and I just want to confirm more), I also changed t he test.

Reply:
I clicked only the last revision in email, and didn't realize there was another related.

Maybe mention in description why it was revert (it could be better if it was on revert PR), and what makes it OK to reland.